### PR TITLE
fix: don't change capitalization of `moduleResolution`

### PR DESCRIPTION
**Pull request by benmccann**
closes https://github.com/svelte-add/svelte-add/issues/404

### CODE REVIEW

File: adders/threlte/config/adder.js [hunk 1]
```diff
@@ -43,7 +43,9 @@ export const adder = defineAdderConfig({
                 if (!data.compilerOptions) data.compilerOptions = {};
 
                 // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                data.compilerOptions.moduleResolution = "Bundler";
+                if (!data.compilerOptions.moduleResolution.toLowerCase() !== "bundler") {
+                    data.compilerOptions.moduleResolution = "bundler";
+                }
```
Comment (AdrianGonz97) [inline adders/threlte/config/adder.js:48]: needs to be tweaked and annotated with ignore lines
```suggestion
                // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
                if (data.compilerOptions.moduleResolution?.toLowerCase() !== "bundler") {
                    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
                    data.compilerOptions.moduleResolution = "bundler";
                }
```

**Review by AdrianGonz97 — CHANGES_REQUESTED** [review]
needs some small adjustments

@manuel3108 can we disable `@typescript-eslint/no-unsafe-member-access` and `@typescript-eslint/no-unsafe-call`? holy cow, are they bothersome in places they really shouldn't be 😅

File: adders/threlte/config/adder.js [hunk 2]
```diff
@@ -43,7 +43,11 @@ export const adder = defineAdderConfig({
                 if (!data.compilerOptions) data.compilerOptions = {};
 
                 // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```
Comment (benmccann) [inline adders/threlte/config/adder.js:45]:
```suggestion
```

**Comment by manuel3108** [comment]
I have disabled the three rules that were bothering us. I did also remove all other comments that were referencing the same issues. We are working with JSON data; we will never be able to work with this data without using any.

File: adders/threlte/config/adder.js [hunk 3]
```diff
@@ -39,11 +39,11 @@ export const adder = defineAdderConfig({
             contentType: "json",
             condition: ({ typescript }) => typescript.installed,
             content: ({ data }) => {
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
                 if (!data.compilerOptions) data.compilerOptions = {};
 
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                data.compilerOptions.moduleResolution = "Bundler";
+                if (!data.compilerOptions.moduleResolution || data.compilerOptions.moduleResolution.toLowerCase() !== "bundler") {
```
Comment (benmccann) [inline adders/threlte/config/adder.js:44]:
```suggestion
                if (data.compilerOptions.moduleResolution?.toLowerCase() !== "bundler") {
```

**Pull request merged**